From d0241e1a50f2bd27876a54ee3047487d7ce66cc5 Mon Sep 17 00:00:00 2001
From: Naveen M K <naveen521kk@gmail.com>
Date: Fri, 18 Jun 2021 12:38:11 +0530
Subject: [PATCH 097/N] Don't use os.pathsep to find EOF not all
 distributions in win32 have them as \ instead check using sys.platform

Signed-off-by: Naveen M K <naveen521kk@gmail.com>
---
 Lib/site.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Lib/site.py b/Lib/site.py
index b344f03..474ef89 100644
--- a/Lib/site.py
+++ b/Lib/site.py
@@ -370,7 +370,7 @@ def setquit():
     The repr of each object contains a hint at how it works.
 
     """
-    if os.sep == '\\':
+    if sys.platform == 'win32':
         eof = 'Ctrl-Z plus Return'
     else:
         eof = 'Ctrl-D (i.e. EOF)'
-- 
2.32.0

